src/OmniSharp.MSBuild/ProjectManager.cs

```diff
@@ -148,14 +151,25 @@ private void ProcessQueue(CancellationToken cancellationToken)
                     projectList.Add(currentProject);
 
                     // update or add project
+                    _failedToLoadProjectFiles.TryRemove(currentProject.FilePath, out int notUsed);
```

Comment:
`out int nonUsed` can just be `out _` in C# 7.

Reply:
Updated